### PR TITLE
Compute instances

### PR DESCRIPTION
**Pull request by iamargus95**
Created the only compute instance required for this project. It's supposed to run Ubuntu-LTS 2004. I have set the default values for the **image** and **size** from this [GIST](https://gist.github.com/mbtamuli/952b2d1ebf940efbfb23aadee91b4faa).

### CODE REVIEW

File: modules/FirewallRules/variables.tf [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+#   type = string
+#   description= "Mention ports."
+#   default = "1194"
+# }
+
+# variable "source_tags" {
+#   type = string
+#   description = "Keep this the same as compute instance name"
+#   default = "vpn-server"
+# }
```
Comment (mbtamuli) [inline modules/FirewallRules/variables.tf]: Don't keep unused code commented.
A trick - If you want it for future reference, commit it once, then remove it in a subsequent commit. Or Notion or something to keep track of these snippets.
```suggestion
```

File: setups/variables.tf [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+variable "region" {
+    type = string
+    description = "Input region. Do not include ZONE here"
+    default = "us-east1"
+}
+
+variable "zone" {
+  type = string
+  description = "Input zone date here. [<region>-<zone>]"
```
Comment (mbtamuli) [inline setups/variables.tf]: `zone date`? :shrug:

File: modules/ComputeInstance_VPN/variables.tf [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+variable "machine_type" {
+  type = string
+  description = "Type of server config"
+  default = "e2-standard-2"
+}
+
+variable "size" {
+  type = string
+  description = "Define the size of your boot disk"
```
Comment (mbtamuli) [inline modules/ComputeInstance_VPN/variables.tf]: Maybe mention the unit in the description - `GB`?

File: modules/ComputeInstance_VPN/main.tf [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+    boot_disk {
+      initialize_params {
+          size = var.size
+          image = var.image
+      }
+    }
+
+    network_interface {
+      network = var.network
+      access_config{}
```
Comment (mbtamuli) [inline modules/ComputeInstance_VPN/main.tf]: What's this empty block doing?

**Review by mbtamuli — CHANGES_REQUESTED** [review]
Overall, looks usable. Have you tested if it's working? 

Bonus tip(read extra work 😅 ): Make a loom video and put it in the PR.

File: setups/variables.tf (hunk 2 above)
Comment (iamargus95) [reply setups/variables.tf]: Supposed to be 'data'. 😅😅

File: modules/ComputeInstance_VPN/variables.tf (hunk 3 above)
Comment (iamargus95) [reply modules/ComputeInstance_VPN/variables.tf]: Got it👍🏽

**Comment by iamargus95** [comment]
Just merged it with the main branch.

**Pull request merged**